src/seml/database.py

```diff
@@ -20,6 +22,19 @@ def get_collection(collection_name, mongodb_config=None, suffix=None):
 
 
 def get_mongo_client(db_name, host, port, username, password, **kwargs):
+    if 'ssh_config' in kwargs:
+        try:
+            from sshtunnel import SSHTunnelForwarder                 
+        except ImportError as e:
+            print('Opening ssh tunnel requires https://sshtunnel.readthedocs.io/en/latest/')
+            raise e
+        server = SSHTunnelForwarder(**kwargs['ssh_config'])
+        server.start()
```

Comment:
How long will this server run?

Reply:
Until they python process is killed... Where would you clean it up?